src/transmitter/index.ts

```diff
@@ -28,6 +31,20 @@ interface HomebaseRequest {
 const upstreamUrl =
   config.INTEGRATION_API || config.DEFAULT_KUBERNETES_UPSTREAM_URL;
 
+let agent = new HttpAgent({
+  keepAlive: true,
+  // Infinity is read as 50 sockets
```

Comment:
Are you sure this is the case? I couldn't find docs that say Infinity === 50, it looks like it really means unlimited sockets 🤔